### DISCUSSION TITLE
Make Model Command -f flag

### DISCUSSION THREAD

**Discussion by funkymonk91** [lunarstorm/laravel-ddd#21] [Ideas]
It'd be nice to be able to do `php artisan ddd:model -f` and have it create the factory in the appropriate directory from the `newFactory` method in the `BaseModel` and have the `protected $model` attribute populated.

**Comment by jaspertey** [comment]
I've thought about this too, definitely an upcoming enhancement to make.

**Comment by jaspertey** [comment]
Implemented in v0.6.0 🚀 - https://github.com/lunarstorm/laravel-ddd/releases/tag/v0.6.0

regarding the `protected $model` property inside the generated factory, that was apparently phased out somewhere along the way from L9. However, if it ends up being needed for the ddd use cases, let me know.

**Reply by jaspertey** [reply]
Followup: looks like the `protected $model` is needed for ddd usage after all... will work on a 0.6.1 to address this.

**Reply by jaspertey** [reply]
All good in 0.6.1 now.

**Reply by funkymonk91** [reply]
Thanks @JasperTey!!! Can't wait to use this today!
